sentence_transformers/backend.py

```diff
@@ -180,32 +182,126 @@ def export_dynamic_quantized_onnx_model(
     if file_suffix is None:
         file_suffix = f"{quantization_config.weights_dtype.name.lower()}_quantized"
 
-    save_or_push_to_hub_onnx_model(
+    save_or_push_to_hub_model(
         export_function=lambda save_dir: quantizer.quantize(quantization_config, save_dir, file_suffix=file_suffix),
         export_function_name="export_dynamic_quantized_onnx_model",
         config=quantization_config,
         model_name_or_path=model_name_or_path,
         push_to_hub=push_to_hub,
         create_pr=create_pr,
         file_suffix=file_suffix,
+        backend="onnx",
     )
 
 
-def save_or_push_to_hub_onnx_model(
+def export_static_quantized_openvino_model(
+    model: SentenceTransformer,
+    quantization_config: OVQuantizationConfig,
+    model_name_or_path: str,
+    push_to_hub: bool = False,
+    create_pr: bool = False,
+    file_suffix: str = "qint8_quantized",
```

Comment:
I would let the user ability provide a dataset which can be just a list of strings. If `dataset=None` you can use SST2 as a default option.